### PR TITLE
Reaction counter

### PR DESCRIPTION
**Pull request by erelaa**
Created a slash command to count and display the total amount of reactions under the message in real-time.

### CODE REVIEW

File: src/slashCommands/reactionCounter.ts [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import { SlashCommandBuilder, EmbedBuilder} from "discord.js";
+import { getThemeColor } from "../functions";
```
Comment (Clodolt) [inline src/slashCommands/reactionCounter.ts:2]: unused import

File: src/slashCommands/reactionCounter.ts [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+import { SlashCommand } from "../types";
+import {CollectorFilter, MessageReaction, User } from "discord.js";
```
Comment (Clodolt) [inline src/slashCommands/reactionCounter.ts:4]: collectorfilter is unused

File: src/slashCommands/reactionCounter.ts [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    try {
+      let reactionCount = 0;
+      const embed = new EmbedBuilder()
+      .setAuthor({name: "Reaction Counter"})
+      .setDescription(`Number of Reactions: ${reactionCount}`)
+
+      const message = await interaction.reply({embeds: [embed], fetchReply:true});
+      const collector = message.createReactionCollector({time: 10000});
+      
+      collector.on('collect', (reaction:MessageReaction, user:User) => {
```
Comment (Clodolt) [inline src/slashCommands/reactionCounter.ts:20]: reaction and user are both unused

File: src/slashCommands/reactionCounter.ts (hunk 1 above)
Comment (erelaa) [reply src/slashCommands/reactionCounter.ts]: fixed

File: src/slashCommands/reactionCounter.ts (hunk 3 above)
Comment (erelaa) [reply src/slashCommands/reactionCounter.ts]: fixed

**Review by Clodolt — APPROVED** [review]

**Pull request merged**